.github/workflows/docs.yml

```
@@ -0,0 +1,42 @@
+          make html
+
+  apidocs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup doc dependencies
+        run: |
+          pip install -f apidocs/requirements.txt
+          pip install -e .
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)